### ISSUE TITLE
Add checkbox aria-label and aria-labelledby attributes

### ISSUE THREAD

**Issue by simeonoff** [IgniteUI/igniteui-angular#858]
### Description
The checkbox component is missing aria-label and aria-labelledby attributes rendering it inaccessible.

### Steps to reproduce  

1. Include igx-checkbox somewhere
2. Add label to the checkbox
3. View the page with a screen reader app

### Result  
The label of the checkbox element is not read

### Expected result  
The labels (visible or not) should be read.
